modules/nw-mutual-tls-auth.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * ingress/ingress-operator.adoc
+
+[id=nw-mutual-tls-auth_{context}]
+= Configuring Mutual TLS Authentication
+
+You can configure the Ingress Controller to enable mutual TLS(mTLS) authentication. mTLS verifies the client certificates (CAs), the certificate revocation list (CRL), and/or any certificate subject filters and rejects requests from clients that do not provide valid certificates.
+
+The {product-title} Ingress Controller configures HAProxy using a configuration template.  The
+template uses environment variables as input parameters. You can configure mTLS using the following environment variables:
```

Comment:
User is just allowed to set `spec.clientTLS` in ingresscontroller to configure mTLS, the ingress operator will watch the `spec.clientTLS` and update those environment variables accordingly. 
Please help check with Devel if the description of these environment variables are required in the document. Thanks.

Reply:
I suspect you're correct. I wondered if this was behind the scenes myself but included it anyways. @Miciah can say for sure.

Reply:
Hongan is right, the environment variables are an implementation detail that isn't necessary to document here.